Assets/Scripts/UI/KeyboardRemap.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using PlayerInput;
+
+public class KeyboardRemap : MonoBehaviour
+{
+    InputPlayer inputPlayer;
+    PlayerInput.PlayerButton action;
+    KeyCode button;
+    IInputPlayer player;
+    int index;
+    public string keyName;
+    bool remaping;
+    
+    public void Update()
+    {
+        if (remaping)
+        {
+            if (Input.anyKey)
+            {
+                foreach (KeyCode vKey in System.Enum.GetValues(typeof(KeyCode)))
+                {
+                    if (Input.GetKey(vKey))
+                    {
```

Comment:
should be get key down, otherwise can cause issues for people with sticky buttons